packages/slice-machine/server/src/api/sentry.ts

```diff
@@ -0,0 +1,41 @@
+// We tunnel the Sentry calls via the express server to avoid add blockers
+// adapted from https://github.com/getsentry/examples/blob/master/tunneling/nextjs/pages/api/tunnel.js
+
+import * as url from "url";
+import express from "express";
+import axios from "axios";
+import { RequestWithEnv } from "./http/common";
+
+// TODO Need these values
+const sentryHost = "";
```

Comment:
these can be inject in the bundler config ;)

Reply:
Updated